vsintegration/src/FSharp.Editor/Options/UIHelpers.fs

```diff
@@ -25,11 +29,33 @@ module internal OptionsUIHelpers =
 
         override this.Child = upcast view.Value
 
+        override this.OnActivate _ =
+            if needsLoadOnNextActivate then
```

Comment:
Do these only occur on the UI thread?
should we use synchronization or perhaps interlocked exchange?

Reply:
@majoch, sweet.  Then we can trust they got it right, or it would have been found and fixed by now :-)